ios/RNSentry.m

```diff
@@ -5,6 +5,16 @@
 #else
 #import "RCTConvert.h"
 #endif
+#if __has_include(<React/RCTBridgeMethod.h>)
```

Comment:
I'm not sure if it's better to import private React headers or use NSSelectorFromString()